var/spack/repos/builtin/packages/rose/package.py

```diff
@@ -68,12 +95,10 @@ class Rose(AutotoolsPackage):
     variant("cxx", default=True, description="Enable c++ language support")
 
     # Use spack install cxxflags=-std=c++11
-    variant("cxx11", default=False, description="Enable c++11 language support")
+    variant("cxx11", default=True, description="Enable c++11 language support")
 
     variant("fortran", default=False, description="Enable fortran language support")
-
-    variant("java", default=False, description="Enable java language support")
-    depends_on("jdk", when="+java")
+    depends_on("openjdk@1.8.0_202-b08", when="+fortran")
```

Comment:
Does it work with any other Java provider? What version of Java is required? You should be able to do something like:
```python
depends_on('java@8', when='+fortran')
```
and have it use any Java 8 provider

Reply:
We have only test with openjdk 1.8 but there is no reason it shouldn't work with other java 8.

Reply:
Changed dependency to be on java@8.